frontend/src/pages/class/[classId]/edit.tsx

```diff
@@ -34,10 +34,10 @@ const EditClass = () => {
       if (klass) {
         await updateClass(klass.id, {
           name: formValues.name,
-          teacherId: klass.teacherId,
+          teacherId: formValues.teacherId,
         });
 
-        router.back();
+        router.push(`/class`);
```

Comment:
why pushing instead of going back after an edit? 🤔

Reply:
This is fixed in the next PR, was a change to make the tests pass more easily ^^